recipes/serd/all/conanfile.py

```diff
@@ -0,0 +1,94 @@
+import os
+
+from conans import ConanFile, tools
+from conans.errors import ConanInvalidConfiguration
+from conans.tools import rmdir
+
+required_conan_version = ">=1.33.0"
+
+
+class Recipe(ConanFile):
+    name = "serd"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://drobilla.net/software/serd.html"
+    description = "A lightweight C library for RDF syntax"
+    topics = "linked-data", "semantic-web", "rdf", "turtle", "trig", "ntriples", "nquads"
+    settings = "build_type", "compiler", "os", "arch"
+    options = {
+        "shared": [True, False],
+        "fPIC": [True, False]
+    }
+    default_options = {
+        "shared": False,
+        "fPIC": True
+    }
+    license = "ISC"
+    exports_sources = "src*", "include*", "doc*", "waf", "wscript", "waflib*", "serd.pc.in"
+
+    def source(self):
+        tools.get(**self.conan_data["sources"][self.version]["serd"],
+                  destination=self.folders.base_source,
+                  strip_root=True)
+        # serd comes with its own modification of the waf build system
+        # It seems to be used only by serd and will be replaced in future versions with meson.
+        # So it makes no sense to create a separate conan package for the build system.
+        from pathlib import Path
+        tools.get(**self.conan_data["sources"][self.version]["autowaf"],
+                  destination=Path(self.folders.base_source).joinpath("waflib"),
+                  strip_root=True)
+
+    def config_options(self):
+        if self.settings.os == 'Windows':
+            del self.options.fPIC
+
+    def configure(self):
+        # its a C library. So C++ properties compiler.{libcxx,cppstd} are not required.
+        del self.settings.compiler.libcxx
+        del self.settings.compiler.cppstd
+        if self.options.shared:
+            del self.options.fPIC
+
+    def validate(self):
+        # disable cross compiling
+        os_build, arch_build, os_host, arch_host = tools.get_cross_building_settings(self)
+        if os_build != os_host or arch_build != arch_host:
```

Comment:
I could not find the docs for this helper, except in the change long... the new way is this guy

```suggestion
        if not tools.cross_building(self):
```

for longevity https://github.com/conan-io/conan/pull/10536

Reply:
I think it should be `if tools.cross_building(self):` (without `not`) because I want to disable it if it uses cross building. I just added it. Tests are fine.

Reply:
I apologize for my lazy copy pasting! I did not even see it